marshal_test.go

```
@@ -172,6 +172,411 @@ func equalStringPointerSlice(leftList, rightList []*string) bool {
+
+			if fastErr != nil || genErr != nil {
+				t.Fatalf("fast err=%v gen err=%v", fastErr, genErr)
+			}
+			if !bytes.Equal(fastData, genData) {
+				t.Errorf("fast path output differs from generic path\nfast: %v\ngen:  %v", fastData, genData)
+			}
+		})
+	}
+}
+
+type namedMapStringString    map[string]string
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(goimports)